transforms/universal/rep_removal/dpk_rep_removal/runtime.py

```diff
@@ -85,7 +85,7 @@ def add_input_params(self, parser: ArgumentParser) -> None:
             "--rep_removal_num_threads",
             type=str,
             required=False,
-            default="4",
+            default=str(cpu_count(logical=False)),
```

Comment:
got for type int and get rid of str()
```suggestion
            default=cpu_count(logical=False),
```